Lib/pyclbr.py

```diff
@@ -45,39 +44,57 @@
 import tokenize
 from token import NAME, DEDENT, OP
 
-__all__ = ["readmodule", "readmodule_ex", "Class", "Function"]
+__all__ = ["readmodule", "readmodule_ex", "Object", "Class", "Function"]
 
-_modules = {}                           # cache of modules we've seen
+_modules = {}  # Initialize cache of modules we've seen.
 
-# each Python class is represented by an instance of this class
-class Class:
-    '''Class to represent a Python class.'''
-    def __init__(self, module, name, super, file, lineno):
+
+class Object:
```

Comment:
I think I need to figure out how to pull in your commit if you would like me to make these changes.